tests/functional/object/mcg/test_s3_vector.py

```
@@ -0,0 +1,206 @@
+        logger.info(f"✓ Vector bucket '{vector_bucket_name}' deleted")
+
+        # Verify bucket is deleted
+        buckets_after_delete = vector_utils.list_vector_buckets(mcg_obj)
+        remaining_buckets = [
+            bucket["VectorBucketName"]
+            for bucket in buckets_after_delete.get("VectorBuckets", [])
+        ]
+        assert (
+            vector_bucket_name not in remaining_buckets
+        ), f"Vector bucket {vector_bucket_name} still exists after deletion"
+        logger.info(f"✓ Verified vector bucket deletion")
```

Ruff (0.15.13)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)